tests/test-switchroot.sh

```diff
@@ -2,6 +2,16 @@
 
 this_script="${BASH_SOURCE:-$(readlink -f "$0")}"
 
+OSTREE_PREPARE_ROOT=$(dirname "${this_script}")/../ostree-prepare-root
```

Comment:
Looking at this again, given that this file is sourced, I'm really not sure I've got this check in the right place?